.github/labeler.yml

```diff
@@ -0,0 +1,3 @@
+# Add 'update-operator' to any changes within 'config/crds' dir
+update-operator:
+  - config/crds/*
```

Comment:
@djwhatle This is something to add into gitignore, I don't remember adding it to my pr.

Reply:
This needs to be checked in to master. It's a github action that auto-labels our PRs, not local config. See https://github.com/konveyor/mig-controller/pull/508